clients/client-python/tests/integration/test_fileset_catalog.py

```diff
@@ -18,110 +17,89 @@
 
 
 class TestFilesetCatalog(IntegrationTestEnv):
-    catalog: Catalog = None
-    metalake: GravitinoMetalake = None
-    metalake_name: str = "testMetalake"
+    metalake_name: str = "testMetalake-TestFilesetCatalog"
     catalog_name: str = "testCatalog"
     schema_name: str = "testSchema"
+
     fileset_name: str = "testFileset1"
-    fileset_alter_name: str = "testFilesetAlter"
+    fileset_comment: str = "fileset_comment"
+    fileset_location: str = "/tmp/TestFilesetCatalog location"
+    fileset_propertie_key1: str = "fileset_propertie_key1"
+    fileset_propertie_value1: str = "fileset_propertie_value1"
+    fileset_propertie_key2: str = "fileset_propertie_key2"
+    fileset_propertie_value2: str = "fileset_propertie_value2"
+    fileset_properties: Dict[str, str] = {fileset_propertie_key1: fileset_propertie_value1,
+                                          fileset_propertie_key2: fileset_propertie_value2}
+    fileset_new_name = fileset_name + "_new"
```

Comment:
It is better that you can add a random prefix to all the names so that you will not fail for any case, like what we did in Java IT?